### PR TITLE
fixed deposit array

### PR DESCRIPTION
**Pull request by viddamao**
deposit for last commit counts the initial balance deposit, fixed that the slope should work ok now.

### CODE REVIEW

File: public/foodpoints.js [hunk 1]
```diff
@@ -65,15 +65,16 @@ $(document).ready(function() {
                     var deposits = [];
 
                     user.trans.forEach(function(exp) {
-                        if (exp.amount > 0) {
+                        if ((exp.amount > 0)&&(exp.amount < 1500)) {
```
Comment (howardc93) [inline public/foodpoints.js]: what's the 1500 mean?

**Comment by viddamao** [comment]
just a value that distinguishes the initial balance adding from other, pushing all positive values to deposit and removing the first would work the same.

File: public/foodpoints.js (hunk 1 above)
Comment (howardc93) [inline public/foodpoints.js]: Ah, 1500 is the maximum that can be added at once?  I'd put that into a constant for improved readability.

**Comment by viddamao** [comment]
Sure, I can fix that. 
Think this value should work since for most cases single time addition for balance will not exceed this amount besides the initial plan adding I guess.
I can change to the second way though, should function the same.

**Comment by howardc93** [comment]
Is this good to merge (tested and everything?)

**Comment by viddamao** [comment]
My local branch worked fine I guess, should be all good.

**Comment by howardc93** [comment]
Do you want me just to merge it, or deploy it to production as well?

**Pull request merged**